contracts/base/Loans.sol

```diff
@@ -361,4 +363,26 @@ contract Loans is LoansInterface, SignerRole {
             liquidated: false
         });
     }
+
+    function getAWholeLendingToken() internal view returns (uint256) {
```

Comment:
as internal functions should `getAWholeLendingToken` and `isCollateralSentEnough` begin with underscores?